src/DotNet.Consolidate/Models/Options.cs

```diff
@@ -6,23 +6,27 @@ namespace DotNet.Consolidate.Models
 {
     public class Options
     {
-        public Options(ICollection<string>? solutions, ICollection<string>? packageIds, ICollection<string>? excludedPackageIds, bool readDirectoryBuildProps, bool reportOverridenDirectoryBuildProps)
+        public Options(ICollection<string>? solutions, ICollection<string>? packageIds, ICollection<string>? excludedPackageIds, string excludedPackageVersionsRegex, bool readDirectoryBuildProps, bool reportOverridenDirectoryBuildProps)
         {
+            Solutions = solutions;
             PackageIds = packageIds;
             ExcludedPackageIds = excludedPackageIds;
-            Solutions = solutions;
+            ExcludedPackageVersionsRegex = excludedPackageVersionsRegex;
             ReadDirectoryBuildProps = readDirectoryBuildProps;
             ReportOverridenDirectoryBuildProps = reportOverridenDirectoryBuildProps;
         }
 
+        [Option('s', "solutions", Required = false, HelpText = "Target solutions for checking. If not specified, all solutions in the working directory will be analyzed.")]
+        public ICollection<string>? Solutions { get; }
```

Comment:
Without the order of the properties and ctor parameters being the same one gets the below exception:

```
Unhandled exception. System.InvalidOperationException: Type DotNet.Consolidate.Models.Options appears to be immutable, but no constructor found to accept values.
   at CommandLine.Core.InstanceBuilder.BuildImmutable[T](Type typeInfo, Maybe`1 factory, IEnumerable`1 specProps, IEnumerable`1 specPropsWithValue, List`1 setPropertyErrors)
   at CommandLine.Core.InstanceBuilder.<>c__DisplayClass1_0`1.<Build>b__5()
   at CommandLine.Core.InstanceBuilder.Build[T](Maybe`1 factory, Func`3 tokenizer, IEnumerable`1 arguments, StringComparer nameComparer, Boolean ignoreValueCase, CultureInfo parsingCulture, Boolean autoHelp, Boolean autoVersion, Boolean allowMultiInstance, IEnumerable`1 nonFatalErrors)
   at CommandLine.Parser.ParseArguments[T](IEnumerable`1 args)
   at DotNet.Consolidate.Program.Main(String[] args) in E:\Projects\Lombiq\dotnet-consolidate\src\DotNet.Consolidate\Program.cs:line 26
```

See: https://github.com/commandlineparser/commandline/wiki/Immutable%20Options%20Type#:~:text=Invalid%20constructors